translations/base-de.yaml

```diff
@@ -627,68 +623,59 @@ storyRewards:
-        desc: >-
-            Du hast es geschafft! Du hast den <strong>Freispiel-Modus</strong> freigeschaltet! Das bedeutet,
-            dass die Formen jetzt <strong>zufällig</strong> erzeugt werden!<br><br>
-
-            Da der Hub ab jetzt einen <strong>Durchsatz</strong> benötigt, empfehle ich dringend eine Maschine zu bauen,
-            die automatisch die gewünschte Form liefert!<br><br>
-
-            Der HUB gibt die gewünschte Form auf der Wires-Ebene aus, also ist alles was du tun musst, sie zu analysieren und
-            automatisch deine Fabrik basierend darauf zu konfigurieren.
-
     reward_blueprints:
-        title: Blaupause
+        title: Blaupausen
         desc: Jetzt kannst du Teile deiner Fabrik <strong>kopieren und
             einfügen</strong>! Wähle ein Areal aus (Halte STRG und ziehe mit
             deiner Maus) und drücke 'C', um zu kopieren.<br><br>Einfügen ist
-            <strong>nicht kostenlos</strong>, du musst
+            <strong>nicht kostenlos</strong>! Du musst
             <strong>Blaupausenformen</strong> produzieren, um die Kopierkosten
             zu decken (Welche du gerade produziert hast)!
-    no_reward:
-        title: Nächstes Level
-        desc: "Dieses Level hat dir keine Belohnung gegeben, aber im Nächsten gibt es
-            eine! <br><br> PS: Denke daran, deine alten Fabriken nicht zu
-            zerstören - Du wirst sie später <strong>alle</strong> noch brauchen,
-            um <strong>Upgrades freizuschalten</strong>!"
-    no_reward_freeplay:
-        title: Nächstes Level
-        desc: Du hast das nächste Level freigeschalten!
-
-    reward_balancer:
-        title: Verteiler
-        desc: Der multifunktionale <strong>Verteiler</strong> wurde freigeschaltet - Er kann
-            benutzt werden, um größere Fabriken zu bauen, indem Gegenstände auf mehrere Fließbänder aufgeteilt und zusammengelegt werden!
-    reward_merger:
-        title: Kompakter Verteiler
-        desc: >-
-            Du hast eine <strong>kompakte Variante</strong> des <strong>Verteilers</strong> freigeschalten - Sie verteilt zwei Fließbänder auf eins!
-    reward_belt_reader:
-        title: Fließband Leser
-        desc: You have now unlocked the <strong>belt reader</strong>! It allows you to
-            measure the throughput of a belt.<br><br>And wait until you unlock
-            wires - then it gets really useful!
     reward_rotater_180:
-        title: Rotater (180 degrees)
-        desc: You just unlocked the 180 degrees <strong>rotater</strong>! - It allows
-            you to rotate a shape by 180 degrees (Surprise! :D)
+        title: Rotierer (180°)
+        desc: Du hast eine weitere Variante des <strong>Rotierers</strong> freigeschaltet! Mit ihm
+            kannst du Formen um 180° drehen (Überraschung! :D).
+    reward_wires_painter_and_levers:
+        title: Wires & Quad Painter
+        desc: "You just unlocked the <strong>Wires Layer</strong>: It is a separate
```

Comment:
Hier fehlt eine übersetzung noch :)

Reply:
Die muss ich noch machen. Aber erst später, habe die nur in Reihe mich base-en.yaml gebracht, damit ich es beim übersetzen einfacher habe